Makefile

```diff
@@ -25,7 +25,7 @@ mypy:
 
 .PHONY: test
 test:
-	coverage run -m pytest --durations=10
+	coverage run -m pytest --durations=10 -W ignore::DeprecationWarning
```

Comment:
we shouldn't ignore all `DeprecationWarning`s, also please move any warning ignores [here](https://github.com/pydantic/pydantic-settings/blob/79571e35277195c7070f40a1bec2ffb60c2454c1/pyproject.toml#L57)